recipes/opensim/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
```

Comment:
Only valid for Linux; is this script used for Mac?

Reply:
Add `openjdk` to the requirements to use the openjdk package from conda-forge